Ex-SCOT-Forecasting-Origin-Team/WhatToEat
# Display Google Map in cuisine.js and nearbySearch the restaurant

## Purpose of this request
1. Display a google map in cuisine.js and mark your current location
2. Search for nearby restaurants and print them to a list

## What is being changed?
1. Remove unused index.html
2. Update README for instructions on how to create a Google location API key 
3. Add a google-maps-react-hooks to package.json
4. Update index.css for the CSS container of the map being displayed
5. Add GoogleMap Tag in the HTML returned by cuisine.js to display google maps
6. Call GoogleMap nearby search API to get a list of nearby restaurants and display it on html.

## Preemptive steps before testing
1. `npm install @ubilabs/google-maps-react-hooks -D`
2. Follow the README to create your Google map API keys.

## Testing
1. `npm start`
2. navigate to the Chinese cuisine page and see the map is being displayed
3. click the button that said nearby restaurant search then verify a list of restaurants exists.
